kousa/lib/socket_handler.ex

```diff
@@ -769,9 +769,9 @@ defmodule Kousa.SocketHandler do
     end
   end
 
-  def f_handler("get_user_profile", %{"userId" => user_id}, %State{} = _state) do
-    user = Beef.Users.get_by_id(user_id)
-
+  def f_handler("get_user_profile", %{"userId" => id_or_username}, %State{} = _state) do
+    user = Beef.Users.get_by_username(id_or_username)
+    user = if is_nil(user), do: Beef.Users.get_by_id(id_or_username), else: user
```

Comment:
Here's how you can check if the input is a `uuid` https://github.com/benawad/dogehouse/blob/staging/kousa/lib/routes/scheduled_room_routes.ex#L15

use that to know if you should lookup by id or username